### DISCUSSION TITLE
NPCs

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#2577] [General]
Hi,
I was wondering if it is possible to also attach conditions to an NPC's vendor shop's stock of items. (Maybe with flagbools)
Could kinda of create NPCs that help someone store something of theirs if they have certain item on them/bool active. Then also help them sell what they placed in there to others.
(Though with my limited knowledge on how NPC vendor works, even if the stock limit can be enforced, the types of items within that vendor can only be changed by physically altering the available item of the vendor .dat files.)

Hmm on further thought, since this (https://github.com/SmartlyDressedGames/U3-Docs/blob/ed2df951c3c7e9b4ac155c836591e0c8ef0cbf11/WeatherAsset.md) is possible, it would be possible to hide certain items in the NPC store as well I think.

Sorry for the ramblings, sorting out my thoughts as well.

**Comment by SDGNelson** [comment]
Sort of: you can use conditions with `Buying_#_Conditions` and `Selling_#_Conditions` (# = index), however the vendors do not have a setup for "rewards" yet.

**Reply by Ongjming** [reply]
I see, thanks! :)
